src/panels/lovelace/editor/hui-dialog-migrate.ts

```diff
@@ -0,0 +1,85 @@
+import { html, LitElement, PropertyDeclarations } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+import "@polymer/paper-button/paper-button";
+import "@polymer/paper-dialog/paper-dialog";
+// tslint:disable-next-line
+import { PaperDialogElement } from "@polymer/paper-dialog/paper-dialog";
+
+import { HomeAssistant } from "../../../types";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { migrateConfig } from "../common/data";
+
+export class HuiDialogMigrate extends hassLocalizeLitMixin(LitElement) {
+  protected hass?: HomeAssistant;
+  private _reloadLovelace?: () => void;
+
+  static get properties(): PropertyDeclarations {
+    return {
+      hass: {},
+      _dialogClosedCallback: {},
+    };
+  }
+
+  public async showDialog({ hass, reloadLovelace }) {
```

Comment:
We should change how we do this, and instead store the params as 1 object instead of many props. 

```ts
public async showDialog(dialogParams: MigrateCardDialogParams) {
  this._dialogParams = dialogParams
```